clients/vss-testclient/requirements.txt

```diff
@@ -0,0 +1,3 @@
+cmd2~=1.3
+websockets~=8.1
+pygments~=2.7
```

Comment:
CQ for these libraries of the specified version are still missing?

Reply:
Submitted.